### PR TITLE
move pipelines and resolvers controller from leader election to state…

### PR DESCRIPTION
**Pull request by jkhelil**
…fulsets

### CODE REVIEW

**Review by khrm — COMMENTED** [review]
/lgtm
/approve

File: components/pipeline-service/development/main-pipeline-service-configuration.yaml [hunk 1]
```diff
@@ -1684,6 +1684,7 @@ spec:
       kube-api-qps: 50
       replicas: 2
       threads-per-controller: 32
+      statefulset-ordinals: true
```
Comment (hugares) [inline components/pipeline-service/development/main-pipeline-service-configuration.yaml:1687]: e2e tests are failing waiting for pipeline service to be "synced" from ArgoCD stand point.
```
Waiting 10 seconds for application sync
pipeline-service-in-cluster-local                  OutOfSync   Healthy
```

**Comment by jkhelil** [comment]
Quick update on this Still trying to understand why pipeline-service is out of sync, i can see the pod running though

**Pull request merged**